curvlinops/kfac.py

```diff
@@ -536,6 +537,10 @@ def _compute_loss_and_backward(self, output: Tensor, y: Tensor):
             output = rearrange(output, "batch ... c -> (batch ...) c")
             y = rearrange(y, "batch ... c -> (batch ...) c")
 
+        # we will differentiate w.r.t. the model parameters to trigger the backward
+        # hooks that compute the grad-output based Kronecker factors
+        params = [p for p in self._model_func.parameters() if p.requires_grad]
```

Comment:
What is the reason not to use `self._params` here?

Reply:
Yap, that's much better!